content/working_with/console/templates-mgmt.md

```diff
@@ -10,7 +10,9 @@ aliases: /manager_webui/templates-mgmt/
 
 In Cloudify’s premium edition, The UI framework allows sys-admins to define what sets of pages the users see when they log into the system, and which widgets these pages contain. Those pre-defined sets of pages are called UI Templates. 
 
-* A UI template applies for specific user-roles, in specific tenants.
+{{% note %}}
+An UI template applies for specific user-roles, in specific tenants.
```

Comment:
It should actually be "A UI", not "An UI"

Reply:
Done.